docs/source/tracking/index.rst

```diff
@@ -0,0 +1,311 @@
+
+.. figure:: ../_static/images/tracking/tracking-metrics-ui-temp.png
+    :align: center
+    :figwidth: 900
+
+    A screenshot of the MLflow Tracking UI, showing a plot of validation loss metrics during model training.
+
+Quickstart
+==========
+If you haven't used MLflow Tracking before, we strongly recommend going through the following tutorial(s) to become familiar with the basic experiment tracking workflow.
+
+.. raw:: html
+
+    <section>
+        <article class="simple-grid">
+            <div class="simple-card">
+                <a href="../getting-started/intro-quickstart/index.html">
+                    <div class="header">
+                        MLflow Tracking Quickstart
+                    </div>
+                    <p>
+                    A great place to start to learn the fundamentals of MLflow Tracking! Learn in 5 minutes how to log, register, and load a model for inference. 
+                    </p>
+                </a>
+            </div>
+            <div class="simple-card">
+                <a href="../getting-started/logging-first-model/index.html">
+                    <div class="header">
+                        Intro to MLflow Tutorial
+                    </div>
+                    <span>
+                        Learn how to get started with the basics of MLflow in a step-by-step instructional tutorial that shows the critical 
+                        path to logging your first model
+                    </span>
+                </a>
+            </div>
+        </article>
+    </section>
+
+Concepts
+========
+
+.. _runs:
+
+Runs
+----
+MLflow Tracking is organized around the concept of *runs*, which are executions of some piece of
+data science code, for example, a single ``python train.py`` execution. Each run records metadata
+(various information about your run such as metrics, parameters, start and end times) and artifacts
+(output files from the run such as model weights, images, etc).
+
+Experiments
+-----------
+An experiment groups together runs for a specific task. You can create an experiment using the CLI, API, or UI.
+The MLflow API and UI also let you create and search for experiments. See :ref:`Organizing Runs into Experiments <organizing_runs_in_experiments>`
+for more details on how to organize your runs into experiments.
+
+
+.. _start-logging:
+
+Tracking Runs
+=============
+
+.. toctree::
+    :maxdepth: 1
+    :hidden:
+
+    tracking-api
+
+`MLflow Tracking APIs <tracking-api.html>`_ provide a set of functions to track your runs. The quickest way to start tracking is to use `Auto-logging <autolog.html>`, a powerful feature that allows you to log metrics, parameters, and models without the need for explicit log statements.
+All you need to do is call :py:func:`mlflow.autolog` before your training code.
+
+    .. code-block:: python
+
+        import mlflow
+
+        mlflow.autolog()
+
+        # Your training code...
```

Comment:
IMO, autolog is an advanced feature. I think we can provide a primitive example here:

```python
with mlflow.start_run():
    mlflow.log_param("p", 1)
    mlflow.log_metric("m", 2)
```

Reply:
I see, tried to sell that convenience of autolog... but yea I can put that example as well on the top as a starter:)